### PR TITLE
Unify how to write whether it is required depends on the conditions

### PR DESCRIPTION
**Pull request by nakabonne**
**What this PR does / why we need it**:
In order to unify with https://github.com/pipe-cd/pipe/pull/2881

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/docs-dev/user-guide/configuration-reference.md [hunk 1]
```diff
@@ -408,7 +408,7 @@ Note: You can get examples for those object from [here](/docs/examples/#ecs-appl
 | provider | string | The unique name of provider defined in the Piped Configuration. | Yes |
 | strategy | string | The strategy name. One of `THRESHOLD` or `PREVIOUS` or `CANARY_BASELINE` or `CANARY_PRIMARY` is available. Defaults to `THRESHOLD`. | No |
 | query | string | A query performed against the [Analysis Provider](/docs/concepts/#analysis-provider). | Yes |
-| expected | [AnalysisExpected](#analysisexpected) | The statically defined expected query result. | No (for the `THRESHOLD`strategy, Yes) |
+| expected | [AnalysisExpected](#analysisexpected) | The statically defined expected query result. | Yes if it's the `THRESHOLD`strategy |
```
Comment (nghialv) [inline docs/content/en/docs-dev/user-guide/configuration-reference.md:411]:
```suggestion
| expected | [AnalysisExpected](#analysisexpected) | The statically defined expected query result. | Yes if strategy is `THRESHOLD` |
```

File: docs/content/en/docs-v0.22.x/user-guide/configuration-reference.md [hunk 2]
```diff
@@ -408,7 +408,7 @@ Note: You can get examples for those object from [here](/docs/examples/#ecs-appl
 | provider | string | The unique name of provider defined in the Piped Configuration. | Yes |
 | strategy | string | The strategy name. One of `THRESHOLD` or `PREVIOUS` or `CANARY_BASELINE` or `CANARY_PRIMARY` is available. Defaults to `THRESHOLD`. | No |
 | query | string | A query performed against the [Analysis Provider](/docs/concepts/#analysis-provider). | Yes |
-| expected | [AnalysisExpected](#analysisexpected) | The statically defined expected query result. | No (for the `THRESHOLD`strategy, Yes) |
+| expected | [AnalysisExpected](#analysisexpected) | The statically defined expected query result. | Yes if it's the `THRESHOLD`strategy |
```
Comment (nghialv) [inline docs/content/en/docs-v0.22.x/user-guide/configuration-reference.md:411]:
```suggestion
| expected | [AnalysisExpected](#analysisexpected) | The statically defined expected query result. | Yes if strategy is `THRESHOLD` |
```

File: docs/content/en/docs/user-guide/configuration-reference.md [hunk 3]
```diff
@@ -408,7 +408,7 @@ Note: You can get examples for those object from [here](/docs/examples/#ecs-appl
 | provider | string | The unique name of provider defined in the Piped Configuration. | Yes |
 | strategy | string | The strategy name. One of `THRESHOLD` or `PREVIOUS` or `CANARY_BASELINE` or `CANARY_PRIMARY` is available. Defaults to `THRESHOLD`. | No |
 | query | string | A query performed against the [Analysis Provider](/docs/concepts/#analysis-provider). | Yes |
-| expected | [AnalysisExpected](#analysisexpected) | The statically defined expected query result. | No (for the `THRESHOLD`strategy, Yes) |
+| expected | [AnalysisExpected](#analysisexpected) | The statically defined expected query result. | Yes if it's the `THRESHOLD`strategy |
```
Comment (nghialv) [inline docs/content/en/docs/user-guide/configuration-reference.md:411]:
```suggestion
| expected | [AnalysisExpected](#analysisexpected) | The statically defined expected query result. | Yes if strategy is `THRESHOLD` |
```

Comment (nakabonne) [reply docs/content/en/docs/user-guide/configuration-reference.md:411]: 👍

**Comment by nakabonne** [comment]
@nghialv Fixed 👍

**Comment by nghialv** [comment]
Nice.
/approve

**Pull request merged**